.github/workflows/unused-dependencies.yml

```diff
@@ -0,0 +1,70 @@
+name: unused-dependencies
+
+on:
+  schedule:
+    - cron: "21 4 * * mon"
+  workflow_dispatch:
+
+jobs:
+  cargo-udeps:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: "read"
+      issues: "write"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@nightly
+      - name: Install cargo-udeps
+        uses: taiki-e/install-action@cargo-udeps
+      - name: Run cargo-udeps, all features
+        run: cargo +nightly udeps --all-features | tee log_all_features.txt
```

Comment:
Hmm, this construct is somewhat suspicious.

Without running the script with `-o pipefail`, this will always return success, e.g.
```bash
#!/usr/bin/env bash
false | cat -
echo $?
```
always outputs `0`, unless `-o pipefail` is set.

It looks like GHA on a Linux runner runs with `-o pipefail` though, IIUC https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell. So no action required, unless it doesn't work.